src/app/pages/assets/asset/asset.component.ts

```diff
@@ -495,4 +493,9 @@ export class AssetComponent implements OnInit {
       }
     });
   }
+
+  private enableForm(): void {
```

Comment:
This function should be replaced with: 
 this.readOnly = (this.dialogMode === DialogMode.VIEW);
 like It's used in CompanyComponent, SiteComponent ..